ansible/roles/compute_init/tasks/main.yml

```diff
@@ -0,0 +1,150 @@
+---
+
+- name: Ensure directories exist
```

Comment:
This is running on group `compute_init` which via `everything` layout defaults to `cluster`.

So you're creating the directories etc. on EVERY node. Whereas we only want to do that on the control node.